src/modules/navigator/rtl.cpp

```diff
@@ -40,29 +40,18 @@
 #include "rtl.h"
 #include "navigator.h"
 
-#include <geo/geo.h>
-#include <mathlib/mathlib.h>
-#include <systemlib/err.h>
-#include <systemlib/mavlink_log.h>
-#include <uORB/topics/home_position.h>
-#include <uORB/topics/vtol_vehicle_status.h>
-#include <uORB/uORB.h>
+#include <cfloat>
 
-static constexpr float DELAY_SIGMA = 0.01f;
+using math::max;
```

Comment:
probably you can't take away `#include <mathlib/mathlib.h>` and use the function defined in it...
or is it still in the include chain?